data/translations/nv/time.json

```diff
@@ -0,0 +1,98 @@
+{
+    "12:00 AM": "오전 12시",
+    "12:15 AM": "오전 12시 15분",
+    "12:30 AM": "오전 12시 30분",
+    "12:45 AM": "오전 12시 45분",
+    "1:00 AM": "오전 1시",
+    "1:15 AM": "오전 1시 15분",
+    "1:30 AM": "오전 1시 30분",
+    "1:45 AM": "오전 1시 45분",
+    "2:00 AM": "오전 2시",
+    "2:15 AM": "오전 2시 15분",
+    "2:30 AM": "오전 2시 30분",
+    "2:45 AM": "오전 2시 45분",
+    "3:00 AM": "오전 3시",
+    "3:15 AM": "오전 3시 15분",
+    "3:30 AM": "오전 3시 30분",
+    "3:45 AM": "오전 3시 45분",
+    "4:00 AM": "오전 4시",
+    "4:15 AM": "오전 4시 15분",
+    "4:30 AM": "오전 4시 30분",
+    "4:45 AM": "오전 4시 45분",
+    "5:00 AM": "오전 5시",
+    "5:15 AM": "오전 5시 15분",
+    "5:30 AM": "오전 5시 30분",
+    "5:45 AM": "오전 5시 45분",
+    "6:00 AM": "오전 6시",
+    "6:15 AM": "오전 6시 15분",
+    "6:30 AM": "오전 6시 30분",
+    "6:45 AM": "오전 6시 45분",
+    "7:00 AM": "오전 7시",
+    "7:15 AM": "오전 7시 15분",
+    "7:30 AM": "오전 7시 30분",
+    "7:45 AM": "오전 7시 45분",
+    "8:00 AM": "오전 8시",
+    "8:15 AM": "오전 8시 15분",
+    "8:30 AM": "오전 8시 30분",
+    "8:45 AM": "오전 8시 45분",
+    "9:00 AM": "오전 9시",
+    "9:15 AM": "오전 9시 15분",
+    "9:30 AM": "오전 9시 30분",
+    "9:45 AM": "오전 9시 45분",
+    "10:00 AM": "오전 10시",
+    "10:15 AM": "오전 10시 15분",
+    "10:30 AM": "오전 10시 30분",
+    "10:45 AM": "오전 10시 45분",
+    "11:00 AM": "오전 11시",
+    "11:15 AM": "오전 11시 15분",
+    "11:30 AM": "오전 11시 30분",
+    "11:45 AM": "오전 11시 45분",
+    "12:00 PM": "오후 12시",
+    "12:15 PM": "오후 12시 15분",
+    "12:30 PM": "오후 12시 30분",
+    "12:45 PM": "오후 12시 45분",
+    "1:00 PM": "오후 1시",
+    "1:15 PM": "오후 1시 15분",
+    "1:30 PM": "오후 1시 30분",
+    "1:45 PM": "오후 1시 45분",
+    "2:00 PM": "오후 2시",
+    "2:15 PM": "오후 2시 15분",
+    "2:30 PM": "오후 2시 30분",
+    "2:45 PM": "오후 2시 45분",
+    "3:00 PM": "오후 3시",
+    "3:15 PM": "오후 3시 15분",
+    "3:30 PM": "오후 3시 30분",
+    "3:45 PM": "오후 3시 45분",
+    "4:00 PM": "오후 4시",
+    "4:15 PM": "오후 4시 15분",
+    "4:30 PM": "오후 4시 30분",
+    "4:45 PM": "오후 4시 45분",
+    "5:00 PM": "오후 5시",
+    "5:15 PM": "오후 5시 15분",
+    "5:30 PM": "오후 5시 30분",
+    "5:45 PM": "오후 5시 45분",
+    "6:00 PM": "오후 6시",
+    "6:15 PM": "오후 6시 15분",
+    "6:30 PM": "오후 6시 30분",
+    "6:45 PM": "오후 6시 45분",
+    "7:00 PM": "오후 7시",
+    "7:15 PM": "오후 7시 15분",
+    "7:30 PM": "오후 7시 30분",
+    "7:45 PM": "오후 7시 45분",
+    "8:00 PM": "오후 8시",
+    "8:15 PM": "오후 8시 15분",
+    "8:30 PM": "오후 8시 30분",
+    "8:45 PM": "오후 8시 45분",
+    "9:00 PM": "오후 9시",
+    "9:15 PM": "오후 9시 15분",
+    "9:30 PM": "오후 9시 30분",
+    "9:45 PM": "오후 9시 45분",
+    "10:00 PM": "오후 10시",
+    "10:15 PM": "오후 10시 15분",
+    "10:30 PM": "오후 10시 30분",
+    "10:45 PM": "오후 10시 45분",
+    "11:00 PM": "오후 11시",
+    "11:15 PM": "오후 11시 15분",
+    "11:30 PM": "오후 11시 30분",
+    "11:45 PM": "오후 11시 45분"
+}
```

Comment:
These times are Korean. Replace with navajo time translations? or if we do not have them use english times.

Reply:
I copied this file over from English. Looks like that one is wrong too. I'll change both to English (I have no idea why English is displaying correctly).
We are using English until we get time back from the translator.